content/en/tracing/troubleshooting/agent_apm_resource_usage.md

```diff
@@ -0,0 +1,40 @@
+---
+title: APM Agent Resource Usage
+kind: Documentation
+aliases:
+  - /tracing/troubleshooting/agent_apm_resource_usage
+---
+
+## How the APM Agent works
+
+The APM Agent is CPU-bound and its CPU usage is correlated with the number of spans received per second.
+
+The APM Agent is designed to buffer unprocessed payloads in memory to avoid data loss. Consequently, throttling the Agent process because of an insufficient CPU limit can lead to out of memory.
```

Comment:
```suggestion
The APM Agent is designed to buffer unprocessed payloads in memory to avoid data loss. Consequently, throttling the Agent process because of an insufficient CPU limit can lead to an out of memory issue.
```